draft-ietf-quic-qlog-quic-events.md

```diff
@@ -881,14 +881,17 @@ QUICDatagramsSent = {
 {: #quic-datagramssent-def title="QUICDatagramsSent definition"}
 
 Since QUIC implementations rarely control UDP logic directly, the raw data
-excludes UDP-level headers in all fields.
-
-The `datagram_id` is a qlog-specific concept to allow tracking of QUIC packet
-coalescing inside UDP datagrams. Since QUIC generates many UDP datagrams, unique
-identifiers are required to be able to track them individually in qlog traces.
-However, neither UDP nor QUIC exchanges datagram identifiers on the wire.
-Selecting identifier values is thus left to qlog implementations, which should
-consider how to generate unique values within the scope of their created traces.
+excludes UDP-level headers in all RawInfo fields.
+
+Multiple QUIC packets can be coalesced in a single UDP datagram, especially
+during the handshake (see {{Section 12.2 of QUIC-TRANSPORT}}). However, neither
+QUIC nor UDP themselves provide an explicit mechanism to track this behaviour.
+Implementations willing to track coalescing across packet-level and
+datagram-level qlog events SHOULD thus use the local and qlog-specific concept
+of a "datagram identifier" in the `datagram_id` field. Selecting specific and
+locally unique `datagram_id` values is left to qlog implementations, but in
+general multiple packet-level events sharing the same `datagram_id` SHOULD
```

Comment:
This second SHOULD kudt seems to be repeating the requirement above?

Reply:
It is; I wasn't aware that's a problem. Would you prefer to change the first `SHOULD` to a `should` or the second one here?

Reply:
I made a suggestion PTAL